scripts/cd/olm-setup.sh

```diff
@@ -274,10 +274,20 @@ replace_with_sed() {
     rm -rf ${TMP_CSV}
 }
 
+check_sha_length() {
+  if [[ "${QUAY_NAMESPACE_TO_PUSH}" == "codeready-toolchain" ]] && [[ ! ${#1} -eq 7 ]]; then
+        echo "ERROR: The number of characters in the short version of SHA is different than 7 characters for the ${2} repository.
+       Images pushed to codeready-toolchain quay org can use only SHA with 7 characters to maintain the dependency tree of the operator updates.
+       Try to use a new fresh clone of the repository instead."
```

Comment:
I wonder if/how using a fresh clone of the repository solves the problem?

Reply:
We thought that in local forks with many branches the output of the commands is different since they count like multiple objects and affect the SHA (@MatousJobanek can correct/add to this), but TBH I haven't found any clear evidence of that on the internet or in the git doc. I would maybe add this explanation to this line of comment but as said I haven't found the evidence of that 🤷‍♂️

Reply:
because it solved the problem for me - after doign a fresh clone in my machine I got 7 chars instead of 8. Based on the git docs it looks like it decides how many chars to use based on the number of objects.